src/main/java/com/redhat/devtools/lsp4ij/settings/jsonSchema/TypeScriptLanguageServerConfigurationJsonSchemaFileProvider.java

```diff
@@ -0,0 +1,32 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at https://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package com.redhat.devtools.lsp4ij.settings.jsonSchema;
+
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * JSON schema file provider for the TypeScript language server.
+ */
+class TypeScriptLanguageServerConfigurationJsonSchemaFileProvider extends LSP4IJJsonSchemaFileProvider {
```

Comment:
This class should not exists. You should load schema by using the name settings.schema.json because the idea is to provide the capability to define a json schema with ui and export your language server to share it.